ads/google/utils.js

```diff
@@ -64,7 +66,7 @@ export function getMultiSizeDimensions(
     }
 
     // Check that secondary size is not larger than primary size.
-    if (!validateDimensions(width, height,
+    if (!isFluid && !validateDimensions(width, height,
```

Comment:
Would it be possible to skip the entire for loop (line 44) if isFluid==true? Or are lines 46-66 indeed necessary in this case?

Reply:
No, we want the loop to execute as usual, but we don't care to enforce the secondary size <= primary size restriction since fluid ads don't have a primary size.

Reply:
Ok, just checking.